packages/shylu/shylu_dd/frosch/src/InterfaceSets/FROSch_DDInterface_decl.hpp

```diff
@@ -149,7 +150,8 @@ namespace FROSch {
     protected:
         
         int communicateLocalComponents(GOVecVecPtr &componentsSubdomains,
-                                       GOVecVec &componentsSubdomainsUnique);
+                                       GOVecVec &componentsSubdomainsUnique,
+                                       UN priorDofsPerNode = 0);
```

Comment:
What is the purpose of `priorDofsPerNode`?

Reply:
See new commit. When starting with OffsetMap (GID not starting at 0) in constructor of an Xpetra::Matrix with underlyinglib=tpetra that is not square (fillComplete called with this OffsetMap as RangeMap and a continuous DomainMap starting at GID 0). Interface will be identified with offset removed if "FROSCH_OFFSET_MAPS" is defined. Otherwise the previous (standard) communicateLocalComponents method will be used.